apps/builder/app/builder/features/sidebar-left/panels/marketplace/card.tsx

```diff
@@ -45,21 +46,34 @@ const imageStyle = css({
         objectFit: "cover",
       },
     },
+    isPlaceholder: {
```

Comment:
hasAsset and isPlaceholder seem like exclusive bools. Maybe replace with single enum variant?

Reply:
true

Reply:
not true
asset | placeholder

😀